x-pack/plugins/event_log/README.md

```diff
@@ -65,21 +84,50 @@ Below is an document in the expected structure, with descriptions of the fields:
     end: "ISO date of end time for events that capture a duration",
     outcome: "success | failure, for events that indicate an outcome",
     reason: "additional detail on failure outcome",
+    // etc
   },
+
+  // Error fields. All of them are supported.
+  // https://www.elastic.co/guide/en/ecs/current/ecs-error.html
   error: {
     message: "an error message, usually associated with outcome: failure",
+    // etc
+  },
+
+  // Log fields. Only a subset is supported.
+  // https://www.elastic.co/guide/en/ecs/current/ecs-log.html
+  log: {
+    level: "info | warning | any log level keyword you need",
+    logger: "name of the logger",
+  },
+
+  // Rule fields. All of them are supported.
+  // https://www.elastic.co/guide/en/ecs/current/ecs-rule.html
+  rule: {
+    author: ["Elastic"],
+    id: "a823fd56-5467-4727-acb1-66809737d943",
```

Comment:
This reminded me of recent `id` vs `rule_id` conversations -- there's not a dedicated spot for us to put `rule_id` (the static id of a rule between clusters), but we can just continue to reference it in the `message` ala:

> [+] Starting Signal Rule execution name: "Persistence via TelemetryController Scheduled Task Hijack" id: "6c25a284-808a-11eb-87f7-018e452bbfc9" rule id: "68921d85-d0dc-48b3-865f-43291ca2c4f2" signals index: ".siem-signals-spong-default"

Reply:
@spong Thank you for pointing to this problem 👍

ECS has a standard [`rule.uuid`](https://www.elastic.co/guide/en/ecs/1.8/ecs-rule.html#field-rule-uuid) field, I'm not sure about its meaning though, the docs are not super clear to me. But even if neither `rule.id` nor `rule.uuid` are suitable for storing a global static rule id, we could do it with a custom field like `kibana.detection_engine.rule.guid`. And then submit a PR to ECS with `rule.guid` field.

I'll need to figure this out - which fields to use for rule ids :)